### ISSUE TITLE
I wrote this on CC website and was told to ask here at GitHub 

### ISSUE THREAD

**Issue by hificom** [uilibrary/Stocky-Issues-and-Feature-request#342]
Feature Request #1.

I request that the main page “DASHBOARD” show overall sales & purchases. not just for the week.

This will give admin a better look at how the company is doing. One week is not enough to gauge and this need to be updated.

I would also suggest that a feature in the SETTINGS all you the option to set your DASHBOARD for it’s view / graphs. Such as view by week, month, year, or all.

9 days ago Flag

Please submit Your feature here : https://github.com/uilibrary/Stocky-Issues-and-Feature-request/issues . Thank you
hifidev
hifidev

hifidev Purchased Supported
9 days ago Flag

Feature request #2

Ability to show, hide, edit, and organize Menu Navigation

A. Show and Hide – Some items are not needed for all users. For example, we don’t need Sales Return and Purchase Return Menu. Those Menu tabs are simple in the way and very frustrating – because we have to scroll pass them. It would be best to just not have them or to just hide those Navigation menu items.

B. Edit – Because we are using English, our language is some what different. We would like a way to Change the language of the menu items. For example:
“People” We would want to change this to “Accounts”
“People > Customer List” – change to “Buyers”
“People > Supplier List” – change to “Suppliers
“People > User List” – Change to “Users/ Admins”

C. Organinze Menu Navigation – we need to be able to change the order of the Menu Navigation.
For example, the current Navigation is:
- Dashboard
- Products
- Adjustment
- Transfer
- Expenses
ect, ect.

We need ability to organize this so we can keep the “Most Used” menu items on top. For example. I would have:

- Dashboard
- Sales
- Purchase
- Reports
- People
- Settings

(I would hide “Doc” because users without Admin control, should not be able to see this.)

Thanks!

**Comment by hificom** [comment]
still waiting on solution.  Thanks!

**Comment by naime-hossain** [comment]
if you are a developer please edit sidebar.vue file and compile the code . if you want to hire our dev for your custom work contact here uilib.help@gmail.com thank you
